include/deal.II/lac/dynamic_sparsity_pattern.h

```diff
@@ -615,6 +615,20 @@ class DynamicSparsityPattern : public Subscriptor
   const IndexSet &
   row_index_set() const;
 
+  /**
+   * Return the IndexSet that contains all column indices stored in this
```

Comment:
I had to think about this a bit because I wasn't clear what "stored" actually means. 

How about this:
```suggestion
   * Return the IndexSet that contains entries for all columns in which at least one element
   * exists in this
```
Same for the next function.